opacus/grad_sample/grad_sample_module.py

```diff
@@ -59,8 +63,24 @@ class GradSampleModule(nn.Module):
     """
     GRAD_SAMPLERS = {}
 
-    def __init__(self, m: nn.Module, *, batch_first=True, loss_reduction="mean"):
+    def __init__(
+        self,
+        m: nn.Module,
+        *,
+        batch_first=True,
+        loss_reduction="mean",
+        strict: bool = True,
```

Comment:
I like this arg name. I'll use this in ModuleValidator too instead of `raise_if_error`! :)